src/test/resources/genesis.it.conf

```diff
@@ -1,25 +1,21 @@
 # Configuration for genesis block generator
 # To generate run from SBT: test:run src/test/resources/genesis.it.conf
 genesis-generator {
-  network-type: "I"
+  network-type: "D"
 
-  initial-balance: 1000000000000000
-  base-target: 200000
-  average-block-delay: 10s
+  initial-balance: 10000000000000000
+  base-target: 100
+  average-block-delay: 60s
   timestamp: 1489352400000 # Comment to use the current time
 
   # seed text -> share
   # the sum of shares should be <= initial-balance
   distributions {
-    "node01": 1400000000000
-    "node02": 1500000000000
-    "node03": 1600000000000
-    "node04": 4000000000000
-    "node05": 5000000000000
-    "node06": 6000000000000
-    "node07": 40000000000000
-    "node08": 50000000000000
-    "node09": 60000000000000
-    "node10": 830500000000000
+    "create genesis wallet devnet-0": 200000000000000
+    "create genesis wallet devnet-0-d": 500000000000000
+    "create genesis wallet devnet-1": 1000000000000000
+    "create genesis wallet devnet-1-d": 1500000000000000
+    "create genesis wallet devnet-2": 2000000000000000
+    "create genesis wallet devnet-2-d": 2500000000000000
```

Comment:
Hm… please rollback these changes, it is for integration tests. But you could create `genesis.dev.conf`